benchmark_runner/common/template_operations/template_operations.py

```diff
@@ -14,6 +15,7 @@
 class TemplateOperations:
     """This class is responsible for template operations"""
 
+    SLEEP_TIME = 3
```

Comment:
Name this something like BULK_SLEEP_TIME so it's more clear what this sleep time is for.  And that should be configurable through a setting, just as thread_limit is.